src/main/java/rip/bolt/ingame/xml/MapFormatXMLParser.java

```diff
@@ -50,4 +57,18 @@ public static TournamentFormat parse(String name) {
         return null;
     }
 
+    public static List<String> getFilesFromFolder(String foldername, String extension) {
```

Comment:
A `MapFormatXMLParser` taking `extension` as a parameter? This simply makes no sense

```suggestion
    public static List<String> getAvailableFormats() {
```
It's not like the `parse` method will let you do anything else in any other folder or with any other filetype